### PR TITLE
Create product_gitlab_roadmapping.md

### PR DESCRIPTION
**Pull request by nick-soutouras**
Adds a new article for Gitlab roadmapping

## 🛠 Summary of changes

Write a brief description of what you changed.
Adds a new page for Gitlab roadmapping for Product folks

### CODE REVIEW

File: _articles/product_gitlab_roadmapping.md
Comment (aduth) [inline _articles/product_gitlab_roadmapping.md:1]: Per build failure, this file will need to be renamed from `product_gitlab_roadmapping.md` to `product-gitlab-roadmapping.md`

File: _articles/product_gitlab_roadmapping.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: Gitlab Roadmapping
```
Comment (aduth) [inline _articles/product_gitlab_roadmapping.md:2]: Minor: GitLab capitalization consistency

```suggestion
title: GitLab Roadmapping
```

File: _articles/product_gitlab_roadmapping.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+description: >
+  Guide to roadmapping in Gitlab
```
Comment (aduth) [inline _articles/product_gitlab_roadmapping.md:4]: Same:

```suggestion
  Guide to roadmapping in GitLab
```

**Review by aduth — APPROVED** [review]
LGTM once build failure is resolved 👍

**Comment by tahineemay** [comment]
Linking Heta's original document: https://docs.google.com/document/d/190jeyVIil9e2FMtcq0t2761yag9XdFz3bq8DM1_TwUo/edit?tab=t.0

**Comment by tahineemay** [comment]
Also, the PPT link for posterity: https://docs.google.com/presentation/d/1ivY3RmbtQJ_Lllc2JwkMpwVey-LEWYm1lswd7muVczI/edit#slide=id.g55dcbb4d19_0_74

File: _articles/product-gitlab-roadmapping.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+Hierarchy: Organize parent epics, child epics, and issues for clarity.
+
+
+## Create a parent Epic
+Navigate to your Group in GitLab.
+Click Epics in the sidebar, then select New Epic.
+Add the following details:
+Title: Name the epic clearly (e.g., Improve Reporting Capabilities).
+Description: Provide a detailed objective or goal.
+Start Date and Due Date: Define the timeframe for the epic.
```
Comment (aduth) [inline _articles/product-gitlab-roadmapping.md:45]: Our spellchecker tool is apparently convinced that "timeframe" is not a word. Maybe it works better in a newer version, else we can add it to our list of ignored misspellings:

https://github.com/GSA-TTS/identity-handbook/blob/fb5957bf60f354c5b3cf17accf006075f218dce2/cspell.json#L4-L59

**Pull request merged**